src/index.ts

```diff
@@ -167,8 +170,7 @@ export async function create (peerId: PeerId, value: CID | PeerId | string, seq:
   // Validity in ISOString with nanoseconds precision and validity type EOL
   const expirationDate = new NanoDate(Date.now() + Number(lifetime))
   const validityType = IpnsEntry.ValidityType.EOL
-  const [ms, ns] = lifetime.toString().split('.')
-  const lifetimeNs = (BigInt(ms) * BigInt(100000)) + BigInt(ns ?? '0')
+  const lifetimeNs = typeof options.lifetimeNs === "bigint" ? options.lifetimeNs : DEFAULT_TTL
```

Comment:
If set and not a bigint, should we error or convert instead of silently overriding?